### PR TITLE
bump babel to v7 & a few other selected dependencies

### PR DESCRIPTION
**Pull request by vicver82**
- `@babel/core`: `6` -> `7`
- `@babel/preset-env`: `1` -> `7`
- `babel-loader`: `7` -> `8`
- `elm`: -> `0.19.0` -> `0.19.0-bugfix2` (for Windows users)
- `elm-hot-webpack-loader`: `0.9` -> `1.0` (for consistency with `elm-webpack-loader` `5.0.0`)
- `webpack-cli`: `3.0` -> `3.1`

I updated `.babelrc` with the new preset and made it not transform ES6 modules by default (a default setting that you see in several webpack starters).

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,3 +1,10 @@
 {
-  "presets": ["env"]
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
```
Comment (simonh1000) [inline .babelrc]: is there a specific reason for this? My feeling is that if the user wants to use modules in their code, then they should think about how (if at all) they want to compile them. I don;t want to pre-empt that choice

Comment (vicver82) [reply .babelrc]: Doing this allows the `@babel/preset-env` preset to transpile to the JS target supported by the desired browser/node environment, _without_ first transpiling libraries & code for the build environment (recent Node.js versions handle ES6 imports/exports).
We can remove this if you think it's better to leave this up to the user (also I might be missing something here).

Comment (vicver82) [reply .babelrc]: Actually Node doesn't fully support ES6 imports/exports. We can remove this setting for general consistency.

**Review by Sebosek — COMMENTED** [review]
What is wrong with this PR?

**Pull request merged**